draft-ietf-ppm-dap.md

```diff
@@ -1556,18 +1545,15 @@ reports in the batch.
 
 ##### Boundary Check
 
-For fixed-size tasks, the batch boundaries are defined by "batch IDs" selected
-by the Leader. Thus the Aggregator needs to check that the query is associated
-with a known batch ID:
-
-* If the query contained in the CollectReq has type `by-batch-id`, the
-  Leader checks that the provided batch ID corresponds to a batch ID it
-  returned in a previous CollectResp for the task.
+For fixed-size tasks, the batch boundaries are defined by numeric batch IDs.
+Thus the Aggregator needs to check that the query is associated with a known
+batch ID:
 
-  [OPEN ISSUE: Consider making batch IDs predictable. See issue #301.]
+* For a CollectReq, the Leader checks that the provided batch ID corresponds to
+  a batch ID it returned in a previous CollectResp for the task.
```

Comment:
Is this required? For this PR we're assuming the mechanism by which batch IDs are assigned is implementation-specific. (This will be addressed in the follow-up PR.)

Reply:
Good point, I think it's not required in this PR. Removed.